olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml

```diff
@@ -319,29 +319,29 @@ spec:
                       - name: "IMAGE_3scale-kourier-control"
                         value: "registry.svc.ci.openshift.org/openshift/knative-v0.18.1:kourier"
                       - name: "IMAGE_eventing-controller__eventing-controller"
-                        value: "registry.svc.ci.openshift.org/openshift/knative-v0.18.6:knative-eventing-controller"
+                        value: "registry.svc.ci.openshift.org/openshift/knative-v0.19.2:knative-eventing-controller"
                       - name: "IMAGE_sugar-controller__controller"
-                        value: "registry.svc.ci.openshift.org/openshift/knative-v0.18.6:knative-eventing-sugar-controller"
+                        value: "registry.svc.ci.openshift.org/openshift/knative-v0.19.2:knative-eventing-sugar-controller"
                       - name: "IMAGE_eventing-webhook__eventing-webhook"
-                        value: "registry.svc.ci.openshift.org/openshift/knative-v0.18.6:knative-eventing-webhook"
-                      - name: "IMAGE_storage-version-migration-eventing-eventing-0.18.6__migrate"
-                        value: "registry.svc.ci.openshift.org/openshift/knative-v0.18.6:knative-eventing-storage-version-migration"
+                        value: "registry.svc.ci.openshift.org/openshift/knative-v0.19.2:knative-eventing-webhook"
+                      - name: "IMAGE_storage-version-migration-eventing-eventing-0.19.2__migrate"
```

Comment:
Do we still need the migrate image? (we don't have a pre-install job anymore do we?)

Reply:
yes, we still build it